shellspy_test.go

```diff
@@ -1,24 +1,57 @@
 package shellspy_test
 
 import (
+	"bytes"
 	"fmt"
+	"os"
 	"os/exec"
 	"shellspy"
+	"strings"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
 )
 
 func TestCommandFromString(t *testing.T) {
 
-	testStrings := []string{"echo", "cat", "ls -la"}
+	input := "echo hello world\n"
+	want := "/bin/echo hello world"
+	got, _ := shellspy.CommandFromString(input)
+
+	if want != got.String() {
+		t.Fatal("something gone wrong")
```

Comment:
Consider using t.Fatalf("want: %q, got: %q", want, got) to explicitly output the expected results vs acutal